Dnn.AdminExperience/ClientSide/Pages.Web/src/services/pageService.js

```diff
@@ -118,7 +118,10 @@ const PageService = function () {
     const toFrontEndPage = function (pageResult) {
         return {
             ...pageResult.page,
-            schedulingEnabled: pageResult.page.startDate || pageResult.page.endDate,
+            schedulingEnabled: pageResult.page.startDate !== null || pageResult.page.endDate !== null,
```

Comment:
This change was not strictly necessary, but React was complaining that `schedulingEnabled` was being assigned a string instead of a boolean value.